back/database/seeds/DatabaseSeeder.php

```diff
@@ -11,6 +11,7 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
+        $this->call(MessagesTableSeeder::class);
```

Comment:
Je préfère que tu fasse cet appel en dernier comme l'ordre est pris en compte dans les seeders. Met `$this->call(MessagesTableSeeder::class);` en dernier dans le fichier